install_script.sh.template

```diff
@@ -1660,6 +1670,28 @@ function update_remote_updates(){
       $sudo_cmd sh -c "sed -i 's/^remote_updates:.*/remote_updates: $remote_updates/' $config_file"
   fi
 }
+function update_installer_registry(){
```

Comment:
nit: could you add a newline before/after the new function?

Reply:
I followed the pattern for the adjacent functions, lmk if you want newlines everywhere, happy to add

Reply:
oh, I assumed it was already the case, my bad. If not then don't bother